Toolchain/nosman/src/nosman/module.rs

```diff
@@ -47,7 +47,8 @@ pub struct InstalledModule {
     pub type_schema_files: Vec<PathBuf>,
     pub module_type: ModuleType,
     #[serde(skip_serializing_if = "Vec::is_empty", default)]
-    pub commands: Vec<NosCommandDesc>
+    pub commands: Vec<NosCommandDesc>,
+    pub dependencies_installed: bool // Whether the module has been loaded with all its static DLL dependencies
```

Comment:
I suppose we don't need this state variable. We could find out if dependency exists in workspace when loading DLLs to register nosman extensions.